scripts/zones/Southern_San_dOria/npcs/Varchet.lua

```diff
@@ -18,7 +18,6 @@ local GAME_TIE = 3;
 -----------------------------------
 
 function onTrade(player,npc,trade)
-
     if (npcUtil.tradeHas(trade, nil, 5)) then
```

Comment:
I need to go for a bit, but I'll switch it to  trade:getGil and trade:getItemCount when I get back.